File: VB/CustomWizardExample/Wizard/Views/IChoosePersonsPageView.vb
Imports DevExpress.DataAccess.Wizard.Views

Namespace CustomWizardExample.Wizard.Views
    Public Enum PersonSourceType
        [Static]
        XmlFilePath
    End Enum
    Public Interface IChoosePersonsPageView
        Inherits IWizardPageView

        Property XmlFilePath() As String
        Property PersonSourceType() As PersonSourceType
        Function ShowWarning() As Boolean
    End Interface
End Namespace

File: CS/CustomWizardExample/Wizard/Views/IChoosePersonsPageView.cs
using DevExpress.DataAccess.Wizard.Views;

namespace CustomWizardExample.Wizard.Views {
    public enum PersonSourceType {
        Static,
        XmlFilePath
    }
    public interface IChoosePersonsPageView : IWizardPageView {
        string XmlFilePath { get; set; }
        PersonSourceType PersonSourceType { get; set; }
        bool ShowWarning();
    }
}
